tests/unit/test_prune.py

```diff
@@ -0,0 +1,256 @@
+from collections import Counter
+from functools import partial
+from unittest import TestCase
+
+import numpy as np
+
+from datumaro.components.annotation import Caption, Label
+from datumaro.components.dataset import Dataset
+from datumaro.components.dataset_base import DatasetItem
+from datumaro.components.media import Image
+from datumaro.components.prune import Prune, match_num_item_for_cluster
+from datumaro.plugins.data_formats.datumaro.exporter import DatumaroExporter
+
+from ..requirements import Requirements, mark_requirement
+
+from tests.utils.test_utils import TestDir
+
+
+class PruneTest(TestCase):
```

Comment:
Use `pytest` for the newly created tests

Reply:
Fix in [4258c20](https://github.com/openvinotoolkit/datumaro/pull/1058/commits/4258c2060f46c723b1ecd46ce603b601bee77ade).